internal/check/milter/milter.go

```diff
@@ -55,6 +55,7 @@ func New(_, instName string, _, inlineArgs []string) (module.Module, error) {
 	case 1:
 		c.milterUrl = inlineArgs[0]
 	case 0:
+		fallthrough
```

Comment:
It is working as intended. See check in Init.

Reply:
Corrected by removing the commit !